src/editors/containers/VideoGallery/index.jsx

```diff
@@ -45,6 +47,15 @@ export const VideoGallery = () => {
     uploadError: messages.uploadVideoError,
   };
 
+  const thumbnailFallback = (
+    <Image
+      thumbnail
+      className="px-6 py-4.5"
+      src={videoThumbnail}
+      style={{ height: '100%' }}
```

Comment:
Maybe we can use https://paragon-openedx.netlify.app/foundations/css-utilities for the classes from here.